terraform/application/workspace_variables/migration.tfvars.json

```diff
@@ -1,12 +1,17 @@
 {
-  "postgres_flexible_server_sku": "GP_Standard_D2ds_v4",
+  "postgres_flexible_server_sku": "GP_Standard_D4ds_v4",
   "app_environment": "migration",
   "cluster": "production",
   "deploy_azure_backing_services": true,
   "enable_monitoring": false,
   "namespace": "cpd-production",
-  "webapp_memory_max": "2Gi",
+  "webapp_replicas": 6,
```

Comment:
We can probably keep the web app replicas/memory as is for now to keep costs down; there's no load on them its just the workers and DB?  one for @cwrw

Reply:
Makes sense.. we just need the DB to get upgraded in ECF. 👍